### ISSUE TITLE
show output for meta: end_play

### ISSUE THREAD

**Issue by hloeffler** [ansible/ansible#19433]
##### ISSUE TYPE

 - Feature Idea

##### COMPONENT NAME

meta

##### ANSIBLE VERSION

```
ansible 2.2.1.0
```

##### OS / ENVIRONMENT

N/A

##### SUMMARY

meta: end_play lacks output

##### STEPS TO REPRODUCE

normal use of meta: end_play

```yaml
    - name: end if checkmode
      meta: end_play
      when: ansible_check_mode

```

##### EXPECTED RESULTS

output (name of the task)

##### ACTUAL RESULTS

no output

```

```

**Comment by ojkastl** [comment]
+1, having some kind of feedback would be nice.

**Comment by bcoca** [comment]
I don't oppose the idea, but it is not on the top of my list to implement, I would go farther and do this in general for all `meta`

**Comment by hloeffler** [comment]
i looked inside the code and was surprised to already find msg= for all meta tasks
so i made a quick fix for -vv output level:
https://github.com/ansible/ansible/pull/20799

**Comment by hloeffler** [comment]
@bcoca please consider my quick-fix PR: https://github.com/ansible/ansible/issues/20799

**Comment by bcoca** [comment]
closing via merged #20799

**Comment by hloeffler** [comment]
@bcoca 
thanks

but i thing the whole output thing needs an overhaul, is there any master issue for it?
